lib/moon/design/form/field/label.ex

```diff
@@ -24,7 +33,16 @@ defmodule Moon.Design.Form.Field.Label do
         ],
         @class
       ])}
-      opts={for: @for}
+      opts={Map.merge(
+        %{
+          for: @for,
+          id: @id,
+          "data-testid": @testid,
+          "phx-click": @on_click,
```

Comment:
```
on_click is event for surface with event name and target. phx-click is only event name?
```

Reply:
using `phx-click` here bc of direct attributes writing - opts works directly this way, not sure surface will replace smth here
`@on_click` already will have live_view's format `%{name: ..., target: ...}` bc it's passed as a property from outside - surface replaces it exactly at the point when attribute is set.